src/Controller/HomeController.php

```diff
@@ -8,10 +8,17 @@
 
 class HomeController 
 {
+    protected $app;
+
+    public function __construct(App $app)
```

Comment:
I would say, the whole `App` object is not needed here. If you look carefully at how you use it, you'll find the minimal required part that needs to be passed to the controller.